ts/parts-3d/Chart3D.ts

```diff
@@ -954,6 +954,9 @@ namespace Chart3D {
             if (e.options.type === 'scatter') {
                 e.options.type = 'scatter3d';
             }
+            if (e.options.type === 'area') {
+                e.options.type = 'area3d';
+            }
```

Comment:
This keeps increasing. Perhaps we should just switch every `xyz` series to `xyz3d` type (e.g. pie)? Do you see any downside, @TorsteinHonsi ?

Reply:
I would be with @pawelfus on this one - e.g. now we are creating support for `polygon3d` that should also be added in here.

Reply:
The difference between `scatter` and `scatter3d` is the data structure. Scatter takes `[x, y]`, while scatter3d takes `[x, y, z]`. But that is not the case with the area/area3d, is it? As far as I understand the data structure is `[x, y]`, but it is rendered in the 3d room, in which case we shouldn't change the `type`.

Reply:
I would agree in general that there are differences between `scatter3d` and `area3d` related to the data structure, however making separate series type for 3d looks to have more benefits:

- keeping core clean and small
- set some basic series options, like `directTouch`
- chance for further improvements, like adding z support in future and much more.

I have made simple test for adding z value to `area3d` and it works well in some cases without any significant work:
<img width="347" alt="Screenshot 2020-06-19 at 15 35 44" src="https://user-images.githubusercontent.com/15050812/85138304-d2a78100-b242-11ea-93f9-b5a996744d81.png">

Reply:
@TorsteinHonsi so would you prefer more the approach shown in the old commit? https://github.com/highcharts/highcharts/pull/13702/commits/547d96e230484d8c0125170ab7e28eec4f298aea

Reply:
In my understanding, these are two different concepts. I don't think an area series with `[x, y, z]` _for each point_ makes much sense. It would be hard to read and understand an area series like that. Instead, I think what we really want is an area to be rendered on a plane within the 3d space. That means equivalent to the current behaviour of column series in 3D space: https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/highcharts/3d/column-zpadding/ . Here, the `groupZPadding` is used to place the are in the Z dimension.

Alternatively, we should probably have a `z` value on the series level, so that it can be placed on the Z axis. I think the current `groupZPadding` option was implemented before we even had the Z axis implementation. Then this option should be available for line, column, 2d-scatter etc.

Reply:
Ok, Thanks for the comment! So I will reverse code to the previous approach with `groupZPadding` and without a need to create new series type. In the meantime, I may look at adding `z` option on the Series level but I would like to deal with it separately of this PR as this might cause some zIndexing calculation issues.